packages/commercetools/helpers/src/index.ts

```diff
@@ -86,6 +91,9 @@ export const getProductAttributes = (products: ProductVariant[], filterByAttribu
 
 export const getProductDescription = (product: ProductVariant): any => (product as any)._description
 
+export const getProductCategories = (product: ProductVariant): string[] => (product as any)._categoriesRef
+
+export const getProductId = (product: ProductVariant): number => (product as any)._id
```

Comment:
I don't think every platform has "id"

Reply:
@filrak but most likely yes (magento, ct, about-you - they all have id), but it could be another identifier, while this helper can have the still same name.